<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabTriggers</class>
 <widget class="QWidget" name="TabTriggers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Seed(s):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="indentation">
      <number>12</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>seed</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>condition</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>z</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Analyze the search condition triggers.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboSeedSource">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Current seed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>From matching seeds list</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushExpand">
       <property name="text">
        <string>Expand all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
